README.md

```diff
@@ -75,5 +74,6 @@ $ make test
 ## Meta
```

Comment:
I get `undefined` error when i click on "Contact Sales"

Reply:
done

Reply:
now im getting: 

Error - 21205
Invalid URL

are you seeing the same thing?

Reply:
I am but when I run the app from localhost (because localhost is not a valid outbound URL). Are you running it from ngrok?